store 模块 空指针的bug

程序正常运行时，出现如下panic 日志信息
panic: interface conversion: interface {} is nil, not *mavl.Tree

goroutine 11623612 [running]:
github.com/bityuan/bityuan/vendor/github.com/33cn/chain33/system/store/mavl.(*Store).Get(0xa4e8200, 0xac99cb0, 0xac99cb0, 0x0, 0x0)
	/mnt/xsTest/src/github.com/bityuan/bityuan/vendor/github.com/33cn/chain33/system/store/mavl/mavl.go:92 +0x3bc
github.com/bityuan/bityuan/vendor/github.com/33cn/chain33/system/store.(*BaseStore).processMessage.func2(0x8a97bce, 0x5, 0x17, 0x0, 0x15e507a, 0x0, 0x8a3e5e0, 0xac99cb0, 0x15573d00, 0xb27b6a0, ...)
	/mnt/xsTest/src/github.com/bityuan/bityuan/vendor/github.com/33cn/chain33/system/store/base.go:105 +0x86
created by github.com/bityuan/bityuan/vendor/github.com/33cn/chain33/system/store.(*BaseStore).processMessage
	/mnt/xsTest/src/github.com/bityuan/bityuan/vendor/github.com/33cn/chain33/system/store/base.go:103 +0x24c

--- a/system/store/mavl/mavl.go
+++ b/system/store/mavl/mavl.go
@@ -88,7 +88,7 @@ func (mavls *Store) Get(datas *types.StoreGet) [][]byte {
 	var err error
 	values := make([][]byte, len(datas.Keys))
 	search := string(datas.StateHash)
-	if data, ok := mavls.trees.Load(search); ok {
+	if data, ok := mavls.trees.Load(search); ok && data != nil {
 		tree = data.(*mavl.Tree)
 	} else {
 		tree = mavl.NewTree(mavls.GetDB(), true)

